README.md

```diff
@@ -269,7 +281,7 @@ process.
 `make terraform` complains 
 
 ```
-Initializing the backend...
```

Comment:
When I asked for `...` to be replaced with `…`, I meant only in the section that you added. Not globally.